appcues/src/main/java/com/appcues/SessionMonitor.kt

```diff
@@ -48,7 +48,10 @@ internal class SessionMonitor(
     }
 
     fun reset() {
+        // reset storage before calling track so it can use updated storage information
```

Comment:
before this change the SessionReset event was being sent with a userId, so I Think it was wrong but would be nice to doublecheck this as well.